src/helpers/buildTransactionsSectionsSelector.tsx

```diff
@@ -65,8 +72,19 @@ const buildTransactionsSections = (
       groupTransactionByDate
     );
     sectionedTransactions = Object.keys(transactionsByDate).map(section => ({
-      data: transactionsByDate[section],
-      renderItem: renderItemElement(TransactionCoinRow),
+      data: transactionsByDate[section].map(txn => ({
+        ...txn,
+        accountAddress,
+        mainnetAddress:
+          accountAssetsData[`${txn.address}_${txn.network}`]?.mainnet_address,
+      })),
+      renderItem: ({ item }: any) => (
+        <FastTransactionCoinRow
+          item={item}
+          onTransactionPress={onTransactionPress}
+          theme={theme}
+        />
+      ),
```

Comment:
We should not use renderItem here, instead we should just add some kind of `type` to each item and decide what component to render based on it